package.json

```diff
@@ -43,6 +45,7 @@
     "history": "^4.7.2",
     "html-react-parser": "^0.13.0",
     "ignore-loader": "^0.1.2",
+    "immer": "^9.0.6",
```

Comment:
I think something like this won't get rid of the old package if they are a dependency imported indirectly by another package right?

We probably need to look at where are these old packages get imported and fix from there

Reply:
yeah I'm not as familiar with NPM. If you specify a requirement in pip/poetry of a sub-dependency, it will enforce what you specify across the board. But I think you're right... b/c I see multiple instances of inmer in the lock file now and some are still out of date :facepalm:

Reply:
NPM is completely different in the dependency management, I haven't tried that, but I think [`overrides`](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#overrides) section will fix it:

```
  "overrides": {
    "foo": "^9.0.6"
  }
```

Reply:
yeah either override those or I can later try if I can upgrade them from the dependency that imports those old packages

is the list in the PR description comperhensive for what we want to update to?